epmon/config.yaml

```diff
@@ -194,6 +194,12 @@ elements:
       - /
       - /kms/list-keys
       - /kms/user-quotas
+  modelarts:
```

Comment:
in the metric processor config there are also defined references for modelartsv2
so either add the epmon queries for modelartsv2 or adjust flag and health metrics to not refer to modelartsv2

Reply:
added modelartsv2 endpoints. Thanks @vladimirhasko.